SpecialEmailPage.php

```diff
@@ -205,6 +205,13 @@ public function send( $send = true ) {
 
 		// Compose the wikitext content of the page to send
 		$title = Title::newFromText( $this->title );
+		
+		// if ( !$title->isContentPage() ) {
+		if ( $title->isSpecialPage() ) {
```

Comment:
Why the comment? Can you not use isContentPage?

Reply:
@thomas-topway-it did you have a look at this again?

Reply:
I've checked, so it depends on the following 

https://doc.wikimedia.org/mediawiki-core/REL1_32/php/DefaultSettings_8php.html#a4dbee32a2ad7f63c5d7295313bd3c0fb

we can keep if `( !$title->isContentPage() ) `{ as long as `$wgContentNamespaces` is filled with the additional namespaces of the wiki, however it should be done dynamically since the namespaces are created or managed by a script, see here 

https://github.com/debtcompliance/mediawiki/blob/development/extensions/DcsNamespaces/DcsNamespaces.php

so the most simple is to keep` if ( $title->isSpecialPage() ) { ` unless you don't want to manage that.